commands/daemon.go

```diff
@@ -238,7 +238,7 @@ Note that jobs are not persisted between restarts of the daemon. See
 
 			node.Override(new(dtypes.Bootstrapper), isBootstrapper),
 			node.Override(new(dtypes.ShutdownChan), shutdown),
-			node.Online(),
+			node.Base(),
```

Comment:
What is the functional difference between the old and new injection?

Reply:
Before: https://github.com/filecoin-project/lotus/blob/v1.11.0/node/builder.go#L457-L473
After: https://github.com/filecoin-project/lotus/blob/v1.13.0/node/builder.go#L223-L235
A name change and a different way of enabling the libp2p bits.